charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -182,10 +182,8 @@ spec:
           periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
           successThreshold: {{ .Values.readinessProbe.successThreshold }}
           timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
-        {{- if .Values.resources }}
         resources:
-{{ toYaml .Values.resources | indent 10 }}
-{{- end }}
+          {{- toYaml .Values.resources | nindent 10 }}
```

Comment:
Using `nindent` will create an empty line after `resources`.
I would rather keep the current style, otherwise, LGTM.

```suggestion
{{- toYaml .Values.resources | indent 10 }}
```

Reply:
@dotdc  - I have reverted to use the current style.  Output is below.

`helm template --values=values.yaml .`
```
        resources:
          {}
        securityContext:
          allowPrivilegeEscalation: false
```
`helm template --values=values.yaml --set resources.limits.memory=100MB .`
```
          successThreshold: 1
          timeoutSeconds: 5
        resources:
          limits:
            memory: 100MB
        securityContext:
          allowPrivilegeEscalation: false
```